stl/inc/type_traits

```diff
@@ -1683,11 +1683,11 @@ struct _Invoke_traits { // selected when _Callable isn't callable with _Args
 };
 
 template <class... _Types>
-struct _Invoke_traits<void_t<decltype(_STD invoke(_STD declval<_Types>()...))>,
+struct _Invoke_traits<void_t<decltype(_STD _Invoker<_Types...>::_Call(_STD declval<_Types>()...))>,
     _Types...> { // selected when _Callable is callable with _Args
-    using type                  = decltype(_STD invoke(_STD declval<_Types>()...));
+    using type                  = decltype(_STD _Invoker<_Types...>::_Call(_STD declval<_Types>()...));
     using _Is_invocable         = true_type;
-    using _Is_nothrow_invocable = bool_constant<noexcept(_STD invoke(_STD declval<_Types>()...))>;
+    using _Is_nothrow_invocable = bool_constant<noexcept(_STD _Invoker<_Types...>::_Call(_STD declval<_Types>()...))>;

```

Comment:
Stylistic issue - we avoid unintentional Argument-Dependent Lookup by explicitly qualifying non-`_Ugly` function calls with `_STD` (which is a macro for `::std::`). That's why we said `_STD invoke` instead of plain `invoke`. However, we don't need to worry about Argument-Dependent Lookup when naming class types, or when calling static member functions of class types - which is the case here.

So, saying `_STD _Invoker` is unnecessary; these three occurrences should simply say `_Invoker`. (But the `_STD declval` at the end of each line should remain; that's a non-`_Ugly` function call, which we conventionally qualify even though there are zero arguments.)

I apologize for not having written our Contribution Guidelines document yet - that will explain this and similar conventions. To save you time, I'll go ahead and push a change to your branch to remove these `_STD` occurrences, then I'll run the tests for your change. Thanks!

Reply:
Thank you